zarnitsa/nlp/DataAugmenterNLP.py

```diff
@@ -166,18 +174,20 @@ def get_most_similar(word, topn=topn):
             by_similarity = sorted(
                 queries, key=lambda w: word.similarity(w), reverse=True
             )
-
+            print([w.lower_ for w in by_similarity[:5]])
```

Comment:
debug?

Reply:
yeah. A similar problem has already been. I will delete this one too